packages/providers/anthropic/src/index.ts

```diff
@@ -41,16 +42,21 @@ export type AnthropicContentBlock = {
   [key: string]: unknown;
 };
 
+export class AnthropicMcpProvider extends BaseMcpProvider<unknown> {
```

Comment:
Are we missing something here? Why is this required if the only thing these provider has is a name?